packages/sandbox/src/components/PreviewDimension.tsx

```diff
@@ -0,0 +1,49 @@
+import { FunctionComponent } from 'react'
+import { PreviewDimensionChangeMessage } from '@storyblok/field-plugin'
+import {
+  FormControl,
+  FormControlLabel,
+  FormLabel,
+  Radio,
+  RadioGroup,
+} from '@mui/material'
+
+export const PreviewDimension: FunctionComponent<{
+  previewDimension: PreviewDimensionChangeMessage['data']
+}> = (props) => {
+  const { previewDimension } = props
+
+  return (
+    <FormControl disabled>
+      <FormLabel id="demo-radio-buttons-group-label">
+        Preview Dimension
+      </FormLabel>
+      <RadioGroup
+        aria-labelledby="demo-radio-buttons-group-label"
+        name="radio-buttons-group"
+        value={previewDimension.tag}
+      >
+        <FormControlLabel
+          value="desktop"
+          control={<Radio />}
+          label="Desktop"
+        />
+        <FormControlLabel
+          value="tablet"
+          control={<Radio />}
+          label="Tablet"
+        />
+        <FormControlLabel
+          value="mobile"
+          control={<Radio />}
+          label="Mobile"
+        />
+        <FormControlLabel
```

Comment:
When custom is checked, I would like to see also the value provided, so it is clear is working as expected.

Here:
<img width="639" height="306" alt="Screenshot 2025-07-23 at 13 16 00" src="https://github.com/user-attachments/assets/b956ca81-606b-4859-939d-cc7a8c9c83c1" />